packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -40,44 +40,51 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
   ///
   /// The [childManager] argument must not be null.
   RenderSliverFixedExtentBoxAdaptor({
+    double itemSpacing: 0.0,
     @required RenderSliverBoxChildManager childManager,
-  }) : super(childManager: childManager);
+  }) : _itemSpacing = itemSpacing, super(childManager: childManager) {
+    assert(itemSpacing != null);
+  }
 
   /// The main-axis extent of each item.
   double get itemExtent;
 
-  /// The layout offset for the child with the given index.
+  /// How much empty space to place between each child in the main axis.
```

Comment:
That's right.  They're padded at the bottom, but I trim off the last padding when computing the overall extent for the list.  I'll add a test that shows that.